### ISSUE TITLE
`replot` does not clear canvas

### ISSUE THREAD

**Issue by esternin** [esternin/eXtrema#53]
`replot` is a command that is typically used to rescale the axes, so that all the data fits (the axes are scaled by the first `graph` command, and all subsequent `graph\overlay` do not rescale the axes, so some data may not fit inside the plot scaled by the first `graph` command. 

Because of that, `replot` needs to *blank* the canvas before replotting.  Right now, it re-plots the new, rescaled graph, right on top of the one that is already there, doubling axes etc. Here's a small sample:

![image](https://user-images.githubusercontent.com/29458452/136466445-5a9af3c6-18e9-4571-91af-eb326c874e43.png)

**Comment by esternin** [comment]
The workaround is to execute  `clear\-replot` every time before `replot`

**Comment by esternin** [comment]
@vadz: I was hoping it was a wxWidgets bug, but the problem persists under wx v.3.2.1.

**Comment by vadz** [comment]
Sorry, I don't think there is any specifically wx-related bug here, why do you think so?

**Comment by esternin** [comment]
Because I was locked into thinking about it as the (sub)window not being painted white, while in fact its was the same stupid typo in the (sub)window coordinate box that showed itself as "erasing the wrong (sub)window".

Fixed by [3b9c58a](https://github.com/esternin/eXtrema/commit/3b9c58a9dea44c359dcd82f009db4145137ae03d).

**Comment by esternin** [comment]
P.S.
With that fixed, gdk errors stopped showing up!  They seemed benign, and I could not decipher that it was writing white background rectangle beyond canvas that was doing it, the error messages were, e.g.:
```
(extrema:1253): Gtk-CRITICAL **: 21:13:15.458: gtk_box_gadget_distribute: assertion 'size >= 0' failed in GtkScrollbar
```
And now they are gone!
